slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/presentation/PresentationService.java

```diff
@@ -0,0 +1,38 @@
+package com.binarystudio.academy.slidez.domain.presentation;
+
+import com.binarystudio.academy.slidez.domain.presentation.model.Presentation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Service
+public class PresentationService {
+
+	private final PresentationRepository presentationRepository;
+
+	@Autowired
+	public PresentationService(PresentationRepository presentationRepository) {
+		this.presentationRepository = presentationRepository;
+	}
+
+	public Presentation add(String name) {
+		Presentation presentation = new Presentation(null, name, "", LocalDateTime.now(), null);
+		return presentationRepository.save(presentation);
+	}
```

Comment:
we should be able to set up link to presentation on it's creation
And dateCreated = dataUpdated on this moment